### ISSUE TITLE
FDISK version 1.3.2?

### ISSUE THREAD

**Issue by micahcochran** [FDOS/fdisk#8]
> Interestingly, I not only have fdisk 1.2.0, 1.2.1 and 1.3.1
> but also fdisk132.zip here (2009-06-08) but we only talk
> about 1.2.1 and 1.3.1 in this thread?
[Source](https://www.bttr-software.de/forum/forum_entry.php?id=17515)  from user mceric on DOS ain't dead forum 12.01.2021

**Comment by PerditionC** [comment]
fdisk 1.3.2 was also just published by Tom - to be merged in and publish a release with binaries

**Comment by andrewbird** [comment]
What's the plan with compilers and the FDOS owned repos? I'm thinking that since utilities like these aren't necessarily too critical for the tightest code and if they can use the simplest memory models, maybe we can build automatically with @tkchia's cross compiler? Watcom's an option of course, but it's a massive install and ia16-gcc is just so easy!

**Comment by PerditionC** [comment]
That's a good question.  I would like to setup automatic builds with ia16-gcc, but release builds I'd prefer done with ow.  Ultimately I want to setup so all programs under FDOS can be built to generate a single full release, but each individual repository I want to have automatically build an update/install zip package.  So gcc is fine for per repository building when possible.  I'm not sure yet how best to structure my whole distro ow builds, each repository having  direct support or a single build repository.  I also am not sure how best to handle cats and other libs shared across multiple utilities.  I suppose each should be in their own repository and maybe use submodules.  I know this is all somewhat off topic.  So far I have been trying to keep the repositories off FDOS the actual base programs, but I guess a few additional non program repositories is most logical.  Cats or really kitten is duplicated way too much.  I'm not sure that any other libs are actually shared, but it's been a while since I compiled many of the programs.

**Comment by andrewbird** [comment]
Yes I'd thought about submodules for kitten, as it seems to be all over the place! What's your policy on build platform, do packages have to be buildable on DOS? Apart from the magnificent collection of DOS era compilers and assemblers, it can be a little awkward to establish a build environment under DOS, and would anyone bother?

**Comment by micahcochran** [comment]
Kitten needs its own repository.  There are several tweaks that could be made to it.   Then add Kitten as a [submodule](https://git-scm.com/docs/git-submodule) under each of the repositories that need it.  That is a great idea.

I really like OW and ia16-gcc are cross compilers, which makes the code more accessible to compile on modern machines.  I've thought about using Travis or Github Actions compiling with the freeware version of Turbo C++ v1.01, but the license makes it a headache with the additional headache of having to run DOS emulation.  Also, I'm not so sure how much of the source code is compatible with TC++ 1.01.  Most of it seems to by default compile with some Borland compiler.

**Comment by andrewbird** [comment]
The kitten submodule has been done for other projects, so if you want to use it that it's possible. Ia16-gcc is now covered in your new ticket. I think you should close this.

**Comment by boeckmann** [comment]
Closing as suggested by @andrewbird
